Proposals/xgov-25.md

```diff
@@ -0,0 +1,40 @@
+---
+period: 7
+title: Notiboy Web3 Chat
+author: AP (@Vidhyanandcs)
+company_name: Securecerts Technologies
+category: dApps
+focus_area: User Onboarding
+open_source: No
+amount_requested: 95700
+status: Draft

```

Comment:
Remember to change this to "Final" if you want your proposal to go through the next voting session.

Reply:
Status final